libraries/AP_Tuning/AP_Tuning.cpp

```diff
@@ -49,7 +49,7 @@ const AP_Param::GroupInfo AP_Tuning::var_info[] = {
 
     // @Param: ERR_THRESH
     // @DisplayName: Controller error threshold
-    // @Description: This sets the controller error threshold above which an alarm will sound and a message will be sent to the GCS to warn of controller instability
+    // @Description: This sets the controller error threshold above which an alarm will sound and a message will be sent to the GCS to warn of controller instability while tuning. The error is the rms value of the P+D corrections in the loop. High values in hover indicate possible instability due to too high PID gains or excessively high D to P gain ratios.-1 will disable this message.
```

Comment:
Do we really sound an alarm?!

Reply:
do not know...dont have buzzers on my planes....